fastlane/Fastfile

```diff
@@ -105,6 +105,9 @@ lane(:bump) do |options|
   # Add version file path to change set
   paths_for_commit << version_file_path
 
+  sh "bundle exec rake generate_team_table"
```

Comment:
I think that the CI wants this to be `sh("bundle exec rake generate_team_table")`